client/src/components/Thread.js

```diff
@@ -1,5 +1,6 @@
-import React from 'react'
-import './Thread.css'
+import React from 'react';
+import './Thread.css';
+import CreateComment from './CreateComment';
```

Comment:
There is a typo on line 16. '<h2 class>' Please delete the word "class" or change it to "className"